packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -689,10 +691,13 @@ class TextSelectionOverlay {
     if (!renderObject.attached) {
       return;
     }
-    final Size handleSize = selectionControls!.getHandleSize(
-      renderObject.preferredLineHeight,
-    );
-    _dragStartPosition = details.globalPosition + Offset(0.0, -handleSize.height);
+
+    // [details.globalPosition] is the drag point, it needs to be shifted
+    // to the text middle point that [RenderEditable] uses to get [TextPosition]
```

Comment:
What exactly is the definition of this text middle point?

Reply:
I change it to text position point and describe of the properties of this point